### DISCUSSION TITLE
When I type the !rank command this error pops up "Cannot assign role newbie as Mee6 Helper does not have the necessary permissions." Please help me.

### DISCUSSION THREAD

**Discussion by AskewCow** [danthe1st/Mee6-bypasser#24] [Support] [answered]
**Describe the question**
A clear and concise description of what you want to know.

**Is your Question related to a problem? Please describe.**

**Additional context**
Add any other context or screenshots about the question here.

**Comment by danthe1st** [comment, marked as answer]
You need to give `Mee6-bypasser` the permission `MANAGE_ROLES`. This role of `Mee6-bypasser` must also be higher than all roles that it should assign.
